circle.yml

```diff
@@ -1,11 +1,3 @@
-checkout:
-  post:
-    - >
-      if [ -n "$CI_PULL_REQUEST" ]; then
-        PR_ID=${CI_PULL_REQUEST##*/}
-        git fetch origin +refs/pull/$PR_ID/merge:
-        git checkout -qf FETCH_HEAD
-      fi
```

Comment:
Discussed with @funkyHat - this apparently causes some issues including security ones.